.gitignore

```diff
@@ -3,3 +3,4 @@
 node_modules/
 dist/
 example/package-lock.json
+example/node_modules/
```

Comment:
This is redundant
On line 3 it is defined without leading slash, so it matches node_modules in any subfolder